asset-store/README.md

```diff
@@ -0,0 +1,100 @@
+# Asset Store
+
+## Overview
+
+This example illustrates how to use the [Asset Store](https://kyma-project.io/docs/1.5/components/asset-store/) to store static webpages.
+
+By default, [Minio](https://min.io/) stores all resources on a cluster, but it also allows you to use different cloud providers. Read the Asset Store [tutorials](https://kyma-project.io/docs/components/asset-store#tutorials-tutorials) for more information.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Kyma](https://kyma-project.io/docs/) as the target deployment environment
+
+## Installation
+
+1. Export a GitHub webpage URL of a ready-to-use webpage.
+
+    Example:
+
+    ```bash
+    export GH_WEBPAGE_URL=https://github.com/pPrecel/simple-page-for-asset-store/archive/master.zip
+    ```
+
+2. Apply a Bucket custom resource (CR):
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: assetstore.kyma-project.io/v1alpha2
+    kind: Bucket
+    metadata:
+      name: pages
+      namespace: default
+    spec:
+      region: "us-east-1"
+      policy: readwrite
+    EOF
+    ```
+
+3. Apply an Asset CR:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: assetstore.kyma-project.io/v1alpha2
+    kind: Asset
+    metadata:
+      name: webpage
+      namespace: default
+    spec:
+      source:
+        url: ${GH_WEBPAGE_URL}
+        mode: package
+      bucketRef:
+        name: pages
+    EOF
+    ```
+
+4. Describe the Asset CR:
+
+    ```bash
+    kubectl describe assets.assetstore.kyma-project.io webpage
```

Comment:
Can you use `kubectl get` with `-o jsonpath` that will combine a link to the `index.html`? Thanks to that step 5 will be obsolete.